ldap/servers/slapd/passwd_extop.c

```diff
@@ -288,8 +288,8 @@ passwd_modify_generate_policy_passwd(passwdPolicy *pwpolicy,
     /* if only minalphas is set, divide it into minuppers and minlowers. */
     if (pwpolicy->pw_minalphas > 0 &&
         (my_policy[idx_minuppers] == 0 && my_policy[idx_minlowers] == 0)) {
-        unsigned int x = (unsigned int)time(NULL);
-        my_policy[idx_minuppers] = slapi_rand_r(&x) % pwpolicy->pw_minalphas;
+        uint64_t x = time(NULL);
```

Comment:
should rather be uint32_t x = (uint32_t)time(NULL)
... slapi_rand_r(&x) ...

Reply:
I think coverity is still going to complain about this change as it wants a uint64_t for the time value.  I will try it, but... we do have some time (13 years) until a change is actually needed :-)  So we could skip this for now.

Reply:
Maybe add a comment saying that we should ignore coverity scan warning (because we do not care about the 64-32 bits values when initializing the random seed)

Reply:
Reverted change and added comments.  Please review...